src/Models/HydrostaticFreeSurfaceModels/single_column_model_mode.jl

```diff
@@ -78,6 +78,9 @@ function update_state!(model::HydrostaticFreeSurfaceModel, grid::SingleColumnGri
     end
 
     update_biogeochemical_state!(model.biogeochemistry, model)
+    
+    compute_tendencies && 
+        @apply_regionally compute_tendencies!(model, callbacks)
```

Comment:
I just noticed that the `SingleColumnGrid` was never computing the tendencies. Is this a bug? Was this mode working? I can add a test for it here

Reply:
Found out the issue, because of this line
https://github.com/CliMA/Oceananigans.jl/blob/5a3d3c92f4e737a199e41be2c95da483f266a9f9/src/Models/HydrostaticFreeSurfaceModels/update_hydrostatic_free_surface_model_state.jl#L28-L30

the `update_state!` in this file was neglected because of the lack of the `compute_tendencies` keyword argument.